### DISCUSSION TITLE
Remap Zoom and Pan funtions

### DISCUSSION THREAD

**Discussion by calvin-1** [freeplane/freeplane#1421] [Ideas]
I am really enjoying learning to use this software, it's helping me lay out and visualize complex ideas. 

I primarily work with AutoCAD and the navigation controls are: 
-middle mouse click (hold) to pan
-mouse scroll to zoom

I often switch between the two programs so having to remember which navigation controls I need to use isn't intuitive. I also think the above navigation method is more fluid and efficient. It would be great to add an option to remap these controls.

If this is already possible then please disregard and kindly point me towards the setting :)

**Comment by sifran-github** [comment]
I just want to point out that I like it the way that it is right now (mouse to move, mouse scroll to scoll down) but if it can be configurable, why not.

**Reply by calvin-1** [reply]
Definitely not suggesting to force people to work the way I do but more flexibility with input options is usually a good thing :)

**Comment by macmarrum** [comment]
Inkscape and GIMP use
- middle mouse click (hold) to pan
- Ctrl+scroll to zoom
- scroll and Shift+scroll to scroll up-down and left-right

I know there is also a request to allow rubber-band selection in Freeplane
- #1218

**Reply by theelectricsoul** [reply]
@calvin-1  @sifran-github 

Here is another approach. This is a demo of a firefox extension called [scrollanywhere](https://addons.mozilla.org/en-US/firefox/addon/scroll_anywhere/)

https://github.com/freeplane/freeplane/assets/10260832/8315e1ed-3201-40bd-b804-0f16cc93c674

**Reply by calvin-1** [reply]
this is in line with what I suggested, although right-click is already well utilized and mmb is usually under-utilized

**Reply by theelectricsoul** [reply]
Clicking the right but it utilized but holding the right button is not utilized at all :) 

Sure middle button can also be used. But I prefer the right button, because my finger is always on it :)

**Reply by calvin-1** [reply]
It is definitely a preference thing. I've spent hundreds of hours in AutoCAD so the muscle memory is well established. Having to pause to think about the controls is what I want to avoid.

**Comment by euu2021** [comment]
Related discussion: #388
